### PR TITLE
Add globs in `jstest` runner

### LINKED ISSUE #289: Add support for globs in `jstest` runner
```yaml
runner: jstest
args:
  - tests/mongo/jstests/core/basic1.js
  - tests/mongo/jstests/core/basic2.js
```

It should be possible to replace that configuration with:
```yaml
runner: jstest
args:
  - tests/mongo/jstests/core/basic*.js
```

There is no shell, so we should do that in Go, using https://pkg.go.dev/path/filepath#Glob and similar functions.

### PR DESCRIPTION
**Pull request by b1ron**
Closes #289.

### CODE REVIEW

File: internal/jstest/jstest.go [hunk 1]
```diff
@@ -29,7 +31,28 @@ func Run(ctx context.Context, args []string) (*internal.TestResults, error) {
...
+
+	files := []string{}
+
+	for _, f := range args {
+		dir, err := os.Getwd()
+		if err != nil {
+			return nil, err
+		}
+
+		matches, err := filepath.Glob(filepath.Join(dir, f))
```
Comment (AlekSi) [inline internal/jstest/jstest.go:43]:
```suggestion
		matches, err := filepath.Glob(dir, f)
```

The current directory is implied

File: internal/jstest/jstest.go [hunk 2]
```diff
@@ -29,7 +31,28 @@ func Run(ctx context.Context, args []string) (*internal.TestResults, error) {
...
+		if err != nil {
+			return nil, err
+		}
+
+		for _, m := range matches {
+			i := strings.LastIndex(m, "/tests/")
```
Comment (AlekSi) [inline internal/jstest/jstest.go:49]: Why we need that?

File: tests/mongo.yml [hunk 3]
```diff
@@ -1,15 +1,14 @@
...
 
 results:
   ferretdb:
     stats:
       expected_fail: 2
-      expected_pass: 1
+      expected_pass: 7
     fail:
       # https://github.com/FerretDB/FerretDB/issues/161
       - tests/mongo/jstests/core/basic1.js
```
Comment (AlekSi) [inline tests/mongo.yml:14]:
```suggestion
      - mongo/jstests/core/basic*.js
```

I think it makes sense to use the same path format as in `args`

File: internal/jstest/jstest.go (hunk 2 above)
Comment (b1ron) [reply internal/jstest/jstest.go]: Fixed

File: tests/mongo.yml (hunk 3 above)
Comment (b1ron) [reply tests/mongo.yml:14]: Fixed

File: internal/jstest/jstest.go (hunk 1 above)
Comment (b1ron) [reply internal/jstest/jstest.go]: Fixed.

**Comment by b1ron** [comment]
I mounted the volume under `/mongo` to make the code cleaner. Might that cause problems with the build given that we clone the repo?

File: internal/jstest/jstest.go [hunk 4]
```diff
@@ -18,18 +18,31 @@ import (
...
 	"log"
 	"os/exec"
+	"path/filepath"
 	"strings"
 
 	"github.com/FerretDB/dance/internal"
 )
 
-func Run(ctx context.Context, args []string) (*internal.TestResults, error) {
+func Run(ctx context.Context, dir string, args []string) (*internal.TestResults, error) {
```
Comment (AlekSi) [inline internal/jstest/jstest.go:27]: `dir` is not used

**Comment by AlekSi** [comment]
I think the problem is that, unlike in `gotest` runner, `cmd.Dir` is never set

File: internal/jstest/jstest.go (hunk 4 above)
Comment (b1ron) [reply internal/jstest/jstest.go]: Yes https://github.com/FerretDB/dance/pull/293/commits/a5ac0416f29843f7a6e037c61533c696b7772d69 added confusion, my bad. I removed `dir` and it works.

**Comment by b1ron** [comment]
> I think the problem is that, unlike in gotest runner, cmd.Dir is never set

As discussed this is not needed as we are in the calling process's current directory and everything works from this directory.

**Comment by AlekSi** [comment]
Added `cmd.Dir = dir` just for consistency

**Review by AlekSi — APPROVED** [review]

**Pull request merged**